app/js/controllers.js

```diff
@@ -1797,8 +1797,31 @@ angular.module('myApp.controllers', ['myApp.i18n'])
         })
       }
       if (selectedMessageIDs.length) {
-        ErrorService.confirm({type: 'MESSAGES_DELETE', count: selectedMessageIDs.length}).then(function () {
-          AppMessagesManager.deleteMessages(selectedMessageIDs).then(function () {
+        var peerString = $scope.curDialog.peer
+        var peerID = $scope.curDialog.peerID
+        var firstChar = peerString.charAt(0)
```

Comment:
Yep, naming can be confusing, it's legacy thing here. Basically:
isMegagroup = supergroup in apps
isBroadcast = channel in apps
isChannel = supergroup or channel (it's because of how it work internally, both have inbox, separated from user's inbox)
isAnyGroup - usual groups and supergroups in app (channels are not included here, since it's not a group)
isUser can be done by comparing peerID > 0